lib/erb_lint/cli.rb

```diff
@@ -261,7 +258,7 @@ def option_parser
           end
         end
 
-        opts.on("--lint-all", "Lint all files matching #{glob}") do |config|
+        opts.on("--lint-all", "Lint all files matching configured glob [default: #{DEFAULT_LINT_ALL_GLOB}]") do |config|
```

Comment:
This is the default when nothing configured. The only thing I'm uncertain of is whether or not it's confusing to show? If yes, perhaps best to not show it at all?